lib/internal/errors.js

```diff
@@ -28,7 +28,6 @@ function makeNodeError(Base) {
     constructor(key, ...args) {
       super(message(key, args));
       this[kCode] = key;
-      Error.captureStackTrace(this, NodeError);
```

Comment:
The frame is not visible because newer v8 versions remove the constructor frame by default.
That's the reason why I removed the `Error.captureStackTrace` call. Do you maybe know a better description for the commit message to make it clearer?